bitbucket-pipelines.dist.yml

```diff
@@ -0,0 +1,127 @@
+---
+
+definitions:
+  installscript: &baseInstallDeps
+    export DEBIAN_FRONTEND='noninteractive' ;
+    export COMPOSER_ALLOW_SUPERUSER=1 ;
+    IPADDRESS=$(grep "$HOSTNAME$" /etc/hosts | grep -Po '^\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3}') ; export IPADDRESS ;
```

Comment:
Using 127.0.0.1 is not gonna work here because on bitbucket everything is docker instance and localhost is not properly mapped to other services. I could replace 0.0.0.0 with explicit correct IP that I detect earlier in the script. The same issue exists in gitlab given that both systems use similar approach to CI implementation.